test/unit/Model.js

```diff
@@ -3565,6 +3565,32 @@ describe("Model", () => {
 					return expect(callType.func(User).bind(User)({"id": 1}, {"data1": "Charlie"})).to.be.rejectedWith("You must update all or none of the combine attributes when running Model.update. Missing combine attributes: data2.");
 				});
 
+				it("Shouldn't throw error if update with property allowing multiple types", async () => {
+					updateItemFunction = () => Promise.resolve({});
+					User = dynamoose.model("User", {"id": Number, "data": [String, Number]});
+
+					await callType.func(User).bind(User)({"id": 1}, {"data": "Bob"});
```

Comment:
Since there arent other unit tests for multiple types, I'd recommend adding one more to make sure that passing a number the resulting sdk expression will reflect into a number. 
So an exact copy of this but of this test but with a number being passed instead.
```
await callType.func(User).bind(User)({"id": 1}, {"data": 2.5});
```

I've actually tested and it works fine, think thought it should be part of the suite.

Reply:
Fixed in 93480f88644683ba71ef31dbcc009b1d1533d639